eras/alonzo/test-suite/src/Test/Cardano/Ledger/Alonzo/Translation/TranslatableGen.hs

```diff
@@ -36,49 +49,65 @@ import Test.QuickCheck (
 import Test.QuickCheck.Gen (Gen (MkGen))
 import Test.QuickCheck.Random (mkQCGen)
 
+data TxInfoLanguage era where
```

Comment:
I think this could also do with some haddock.

Reply:
I have a separate plan for PlutusContext golden tests, which will lead to this type disappearing, so I won't bother documenting it now.

Reply:
This will be done as part of this ticket: https://github.com/input-output-hk/cardano-ledger/issues/3923